convert2rhel/pkgmanager/handlers/dnf/__init__.py

```diff
@@ -156,14 +162,26 @@ def _resolve_dependencies(self):
             self._base.resolve(allow_erasing=True)
         except pkgmanager.exceptions.DepsolveError as e:
             loggerinst.debug("Got the following exception message: %s" % e)
-            loggerinst.critical("Failed to resolve dependencies in the transaction.")
+            loggerinst.critical_no_exit("Failed to resolve dependencies in the transaction.")
+            raise exceptions.CriticalError(
+                id_="FAILED_TO_RESOLVE_DEPENDENCIES",
+                title="Failed to resolve dependencies.",
+                description="During package transaction the package manager failed to resolve the necessary dependencies needed for a package replacement.",
```

Comment:
```suggestion
                description="During package transaction, dnf failed to resolve the necessary dependencies needed for a package replacement.",
```

I think its safe to use `dnf` here as it will never have anything related to yum in this part of the code

Reply:
I tried to make the message the same in both dnf and yum files, you think I should change it?

Reply:
It would be better. I think they deserve a message on their own terms. User will only see the yum message if they run a system with yum backend, and dnf if the system has the dnf backend. 

Making it generic here doesn't give much information as they might not know that if it was yum or dnf executing (a bit difficult use case, I know, but who knows...)